TEI.js

```diff
@@ -109,6 +109,29 @@ var exportedXMLIds = {};
 var generatedItems = {};
 var allItems = {};
 
+// replace formatting with TEI tags
+function replaceFormatting (title){
+    var titleText = title;
+    // italics
+    titleText = titleText.replace(/<i>/g, '<hi rend="italics">');
+    titleText = titleText.replace(/<\/i>/g, '</hi>');
+    // bold
+    titleText = titleText.replace(/<b>/g, '<hi rend="bold">');
+    titleText = titleText.replace(/<\/b>/g, '</hi>');
+    // subscript
+    titleText = titleText.replace(/<sub>/g, '<hi rend="sub">');
+    titleText = titleText.replace(/<\/sub>/g, '</hi>');
+    // superscript
+    titleText = titleText.replace(/<sup>/g, '<hi rend="sup">');
+    titleText = titleText.replace(/<\/sup>/g, '</hi>');
+    // small caps
+    titleText = titleText.replace(/<span style="font-variant:small-caps;">|<span style="font-variant: small-caps;">|<sc>/g, '<hi rend="smallcaps">');
```

Comment:
let's go with
```
titleText = titleText.replace(/<span style="font-variant:\s*small-caps;">(.*?)<\/span>/g, '<hi rend="smallcaps">$1</hi>');
titleText = titleText.replace(/<sc>/g, '<hi rend="smallcaps">');
titleText = titleText.replace(/<\/sc>/g, '</hi>');
```
to avoid replacing some unrelated `</span>` tags, though that shouldn't be a frequent occurrence.
Same for `nocase` below